DependencyInjection/Configuration.php

```diff
@@ -81,11 +81,17 @@ public function getConfigTreeBuilder()
                                  ->end()
                             ->end()
                         ->end()
-                        
+
                     ->end()
                 ->end()
                 ->end();
-        
+
+        $rootNode->
```

Comment:
Why adding a new node? It'd make sense to put a `controller` key under `registration` instead